src/frontend/src/scheduler/local.rs

```diff
@@ -42,15 +42,26 @@ pub struct LocalQueryExecution {
     query: Query,
     front_env: FrontendEnv,
     epoch: Option<u64>,
+
+    current_database: String,
+    current_user: String,
```

Comment:
As a better practice, I would suggest to put these two fields into anther construct in `FrontendEnv`, for example, `AuthContext`

Reply:
I introduced an `AuthContext` in `SessionImp` and passing it around. Put it into `FrontendEnv` is not quit reasonable, cuz all fields inside it are stateless.